frontend/__tests__/components/Button/button.test.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { render, screen, renderer } from "../../../test-utils";
+import Button from "../../../components/Button/Button";
+
+describe("Button", () => {
+  it("should render button component", () => {
+    render(<Button>big red button</Button>);
```

Comment:
Once again, not enough testing - the button accepts props, this needs to be tested.

Reply:
I added a test for the button's onClick listener. Hopefully, that will suffice

Reply:
Please re-examine your tests